ocs/README.md

```diff
@@ -9,10 +9,37 @@ state of Ceph-CSI as used in the OpenShift Container Storage product.
 
 This GitHub repository contains branches for different product versions.
 
+## Backports
+
+All changes in this repository are *backports* from the [upsteam
+project][upstream-ceph-csi]. There should be no functional changes (only
+process/CI/building/..) in this repository compared to the upstream project.
+Fixes for any of the release branches should first land in the master branch
+before they may be backported to the release branch. A backport for the oldest
+release should also be backported to all the newer releases in order to prevent
+re-introducing a bug when a user updates.
+
+### Sync `master` with upstream `ceph/ceph-csi:devel`
+
+Syncing branches (including the `master` branch) from upstream should be done
+with a Pull-Request. To create a PR that syncs the latest changes from
+`ceph/ceph-csi:devel` into the `master branch`, [click here][sync-pr].
+
+### Backporting changes from the `master` to `release-*` branches
+
+Once a PR has been merged in the master branch that fixes an issue. A new PR
+with the backport can be created. The easiest way is to use a command like
```

Comment:
Dont we need 'release specific bug' in the cherrypick PR ?

Reply:
yes, most likely. The bot to verify that has been configured with openshift/release#16722 . I do not think we need to document it for now. If we notice the comments from the bot are not helpful enough, we can add a more complete description.

Reply:
Some PRs/Fixes may be release specific, for example it may not be a bug in master at all but in a specific release. Not sure how to tackle it, while we make picking from master made compulsory for the fixes. @nixpanic

Reply:
hmmmm . Looks like @agarwal-mudit already sent lgtm at same time of the comment :)

Reply:
I expect that a fix is still present in upstream for the vast majority of changes to release branches. Maybe not in the `devel` branch, but at least in one of the versions that are available.

But, there probably will always be exceptions. That should not be a problem, it would just need a good explanation in the PR description.